pkg/controller/controller.go

```diff
@@ -109,7 +177,7 @@ func (c *Controller) ProcessNextItem() bool {
 	glog.Infof("processNextItem: next pvc to process: %s\n", key)
 
 	// all checks have passed, let's process it!
-	if err := c.processItem(pvc); err != nil {
+	if err := c.processPvcItem(pvc); err != nil {
```

Comment:
this should be `== nil`, right?

Reply:
yes.  That's still in master, I'll open a small fix-up pr to change it